<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyDRS Enabled Maintenance Interface</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,1,0,0,1">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>IP/Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipLineEdit">
        <property name="placeholderText">
         <string>10.0.6.99/COM1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Port/Baudrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portLineEdit">
        <property name="placeholderText">
         <string>5000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Connection:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="conTypeLabel">
        <property name="text">
         <string>Disconnected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="detailsGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Power Supply</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1,4,1,1,0,1,4,1">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Model:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="psModelLabel">
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>State:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="stateLabel">
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Lock:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lockLabel">
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Select Power Supply:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="addressCombobox"/>
         </item>
         <item>
          <widget class="QPushButton" name="lockButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>res/lock-open.png</normaloff>res/lock-open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>portLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>connectButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ipLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>connectButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
